Cargo.toml

```diff
@@ -20,5 +20,9 @@ include = [
 edition = "2018"
 
 [dependencies]
-embedded-hal = "0.2"
+embedded-hal = { version = "0.2.2", features = ["unproven"] }
 nb = "0.1.1"
+
+[patch.crates-io]
+embedded-hal = { git = "https://github.com/rust-embedded/embedded-hal" }
```

Comment:
Do you know when this will be released? I think the merge should probably wait until we don't have to patch dependencies with git master references.

Reply:
I think we're waiting on more thoughts and reviews on https://github.com/rust-embedded/embedded-hal/pull/127, at which point we should be able to push a minor release.
I bumped it for more reviews today, will follow up in a couple of days once anyone else has had a chance to comment.

Reply:
Sounds good!

Reply:
we've pushed 0.2.3-rc.1 which contains the new digital traits, and i've updated this PR to point to it.
how do y'all feel about depending on an -rc crate?

Reply:
Would this play well with drivers depending on `embedded-hal = "0.2"` ? Would cargo pick the rc version for everybody or would we then need to do the semver trick thing?

Reply:
it turns out rc versions are also cursed with incompatibility...

we've shipped 0.2.3 now and i've updated this to match so i think this is all resolved.